src/openms/source/CHEMISTRY/ElementDB.cpp

```diff
@@ -63,72 +63,72 @@ namespace OpenMS
     return db_;
   }
 
-  const Map<String, const Element*>& ElementDB::getNames() const
+  const map<string, const Element*>& ElementDB::getNames() const
   {
     return names_;
   }
 
-  const Map<String, const Element*>& ElementDB::getSymbols() const
+  const map<string, const Element*>& ElementDB::getSymbols() const
   {
     return symbols_;
   }
 
-  const Map<UInt, const Element*>& ElementDB::getAtomicNumbers() const
+  const map<unsigned int, const Element*>& ElementDB::getAtomicNumbers() const
   {
     return atomic_numbers_;
   }
 
-  const Element* ElementDB::getElement(const String& name) const
+  const Element* ElementDB::getElement(const string& name) const
   {
-    if (names_.has(name))
+    if (names_.count(name) == 1)
```

Comment:
rather ineffective.
This searches the entry, and then discards it again, so `names_.at()` basically has to do the same search again.
Better:
```
auto entry = names_.find(name);
if (entry != names_.end())
{
 ...
}```

Reply:
or with C17 the scope of the iterator can be reduced
```
if (auto entry = names_.find(name); entry != names_.end())
{
}
```

Reply:
You both are right! I'll quickly change it.